resources/windows/wix/service.component.wxs.erb

```diff
@@ -0,0 +1,58 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+  <%- get_services.each do |service| -%>
+    <%- service.options.each do |component| -%>
+    <DirectoryRef Id="<%= component[:directory_ref] %>">
+      <Component Id='<%= component[:id] %>' Guid="<%= component[:guid] %>" <%= "Win64=\"$(var.Win64)\"" unless component[:win64].nil? %>>
+        <%- if component[:file][:id] == "NSSM" -%>
+        <CreateFolder />
```

Comment:
CreateFolder is only added if we expect the folder to not have contents at install time, but later on. MSI won't create an empty folder without it. Will need to look at original hierarchy to decide if it is really necessary.